modules/rasBidAdapter.md

```diff
@@ -1,33 +1,51 @@
 # Overview
 
-Module Name: RingierAxelSpringer Bidder Adapter
+```
+Module Name: Ringier Axel Springer Bidder Adapter
 Module Type: Bidder Adapter
-Maintainer: adp-support@ringieraxelspringer.pl
+Maintainer: support@ringpublishing.com
+```
 
 # Description
 
-Module that connects to RingerAxelSpringer demand sources.
+Module that connects to Ringer Axel Springer demand sources.
 Only banner format is supported.
 
 # Test Parameters
 ```js
-var adUnits = [
-  {
-    code: 'test-div-ad',
-    mediaTypes: {
-      banner: {
-        sizes: [[300, 250], [300,600]]
-      }
-    },
-    bids: [{
-      bidder: 'ringieraxelspringer',
-      params: {
-        area: 'NOWASG',
-        site: 'GLOWNA',
-        network: '1746213',
-        slot: 'top'
-      }
-    }]
-  }
-];
+var adUnits = [{
+  code: 'test-div-ad',
+  mediaTypes: {
+    banner: {
+      sizes: [[300, 250], [300, 600]]
+    }
+  },
+  bids: [{
+    bidder: 'ringieraxelspringer',
```

Comment:
tutaj miał być `ras`

Reply:
a nie w tym nowym adapterze?

Reply:
no ale to jest dokumentacja właśnie pod ten nowy adapter